bin/libast_prereq.sh

```diff
@@ -15,7 +15,7 @@ PATH=$bin_dir:$PATH
 # Legacy build system keeps files generated by iffe with different names under different directories
 # Each array listed below is processed separately and headers generated by iffe are kept in separate directories
 # in a way that is compatible with legacy build system
-iffe_tests=( 'api' 'aso' 'ccode' 'common' 'float' 'fs' 'iconv' 'lib' 'locale' 'mmap' 'ndbm' 'options' 'prog' 'sched' 'sfio' 'siglist' 'sizeof' 'spawn' 'standards' 'sys' 'sys_calls' 'time' 'tmlib' 'tty' 'tv' 'tvlib' 'vfork' 'vmalloc' 'wait')
+iffe_tests=( 'api' 'aso' 'ccode' 'common' 'float' 'fs' 'iconv' 'lib' 'syslocale' 'mmap' 'ndbm' 'options' 'prog' 'sched' 'sfio' 'siglist' 'sizeof' 'spawn' 'standards' 'sys' 'sys_calls' 'time' 'tmlib' 'tty' 'tv' 'tvlib' 'vfork' 'vmalloc' 'wait')
```

Comment:
Meson does not honor order of include paths and this feature test was clashing with another test with same name under ksh93 directory. So I changed it's name.